app/src/main/java/com/vero/tasky/agenda/domain/model/Event.kt

```diff
@@ -0,0 +1,14 @@
+package com.vero.tasky.agenda.domain.model
+
+data class Event(
```

Comment:
Reminder, Task and Event all share some variables

Maybe you can add a parent class, and inherit from that